common/src/main/scala/com/gu/stripe/Stripe.scala

```diff
@@ -48,24 +48,24 @@ object Stripe {
     implicit val codec: Codec[Card] = deriveCodec
   }
 
-  case class Card(id: String, `type`: String, last4: String, exp_month: Int, exp_year: Int, country: String) extends StripeObject {
-    val issuer = `type`.toLowerCase
+  case class Card(id: String, brand: String, last4: String, exp_month: Int, exp_year: Int, country: String) extends StripeObject {
```

Comment:
Should we change this class to `Source` so that `customer.sources` becomes a `StripeList[Source]` do you think?

Reply:
I figured keeping it Card was OK since the sources in practice are still cards but quite happy to call it Source like the Stripe API does.

However this reminds me -- there is an [existing case class Source](https://github.com/guardian/support-workers/blob/master/common/src/main/scala/com/gu/stripe/Stripe.scala#L71) and I could not figure out where it was being used (other than within case class Charge, which in turn I couldn't see where it was used). I think I should understand this before I can rename Card to Source. Can you point me in the right direction for that?

Reply:
Yes it looks as though you could get rid of those 2 classes, I've just tried it and it seems ok. There are a few failing tests but they fail on master as well, I think it's because of the Salesforce work and a data clearout that Jacob did from Zuora.

Reply:
Thanks for checking that. I'm going to rename Card to Source then.

Reply:
@rupertbates @lmath re the failing tests - yes, that's why they're broken. I'm working with Paul to re-configure Zuora UAT. When that's done I'll create some new stub data and fix the tests.